src/helpers/channels/delete_stage_instance.ts

```diff
@@ -0,0 +1,27 @@
+import { cacheHandlers } from "../../cache.ts";
+import { rest } from "../../rest/rest.ts";
+import { ChannelTypes } from "../../types/channels/channel_types.ts";
+import { Errors } from "../../types/discordeno/errors.ts";
+import { endpoints } from "../../util/constants.ts";
+import { requireBotChannelPermissions } from "../../util/permissions.ts";
+
+/** Deletes the Stage instance. Requires the user to be a moderator of the Stage channel. */
+export async function deleteStageInstance(channelId: bigint) {
+  const channel = await cacheHandlers.get("channels", channelId);
+  if (!channel) throw new Error(Errors.CHANNEL_NOT_FOUND);
```

Comment:
Do not throw an error if the channel is not cached instead use `if (channel) // do checks`